### ISSUE TITLE
Name is not defined for "Remove user" button for people picker elements.

### ISSUE THREAD

**Issue by SyedRehmanRazvi** [microsoft/fluentui#12406]
User Experience:
Users who depends on screen reader are facing difficulty as name is not defined for "People picker" remove button.

Repro Steps:
Step 1: Open https://fabricweb.z5.web.core.windows.net/oufr/7.105.2/#/examples/peoplepicker 
Step 2: Search for any user in the people picker search edit field.
Step 3: Navigate to "Remove" button and observe the narrator narration.

Actual Result:
Narrator is announcing as "Remove" button when focus is on remove user button.

Expected Result:
Narrator should announce as "Remove <user> button.

[People Picker.zip](https://github.com/microsoft/fluentui/files/4375051/People.Picker.zip)

**Comment by aneeshack4** [comment]
@SyedRehmanRazvi This is by design - Narrator reads out whatever string is passed into `removeButtonAriaLabel` and it was given the string "Remove". I saw in your video the person said it should say "Remove + the name of the person" - which accessibility guidance says this is necessary? I can see that it could be more helpful - @joschect what do you think? Maybe in the PeoplePicker examples the remove button aria label's string can have each persona name concatenated? It would have involve some logic as it looks like the personas list is prepared separate from the construction of the picker itself. If you know the best way to do this, please suggest it.

**Comment by joschect** [comment]
@aneeshack4 That is correct. I think that perhaps the fix should be that the remove button shouldn't be focusable but I'm not sure about that.

**Comment by ashish315** [comment]
@aneeshack4 Why can't we use custom aria label for "X" button. You can take reference from below screenshot
![image](https://user-images.githubusercontent.com/60913131/96361811-a21b1980-1146-11eb-9080-7521dc03ae79.png)

**Comment by joshmccall221** [comment]
You need to specify  `removeButtonAriaLabel`:
```
    <CompactPeoplePicker
        removeButtonAriaLabel={'Remove'} 
  ```
[see issue #14776:](https://github.com/microsoft/fluentui/issues/14776#issuecomment-768607215)
